src/libs/actions/Report.js

```diff
@@ -265,6 +265,29 @@ function updateIOUReportData(chatReport) {
     });
 }
 
+/**
+ * Get the report ID for a chat report for a specific
+ * set of participants.
+ *
+ * @param {String[]} participants
+ * @returns {Promise} chatReportID
+ */
+function findChatReportID(participants) {
+    if (participants.length < 2) {
+        throw new Error('fetchOrCreateChatReport() must have at least two participants');
```

Comment:
The error message should be updated to say 'findChatReportID() must have at least two participants'.